config.go

```diff
@@ -407,18 +407,20 @@ func loadConfig() (*config, error) {
 				minTimeLockDelta)
 		}
 
-		// Multiple networks can't be selected simultaneously.  Count
+		// Multiple networks can't be selected simultaneously. Count
 		// number of network flags passed; assign active network params
-		// while we're at it.
+		// their bitcoin counterpart before applying ltcParams to them.
 		numNets := 0
 		var ltcParams litecoinNetParams
 		if cfg.Litecoin.MainNet {
 			numNets++
 			ltcParams = litecoinMainNetParams
+			activeNetParams = bitcoinMainNetParams
 		}
 		if cfg.Litecoin.TestNet3 {
 			numNets++
 			ltcParams = litecoinTestNetParams
+			activeNetParams = bitcoinTestNetParams
```

Comment:
If you look below, we apply the Litecoin params to activeNetParams below.

Reply:
Have I misunderstood something? This line is just to prevent equality [here](https://github.com/lightningnetwork/lnd/blob/237ed4fc6ec3f34a67a65e1e79cb43302b0cb634/rpcserver.go#L1200). I am unable to actually test it on main-net, but flipping this setting(faking mainnet for the part) shows testnet: false, as expected...